### PR TITLE
Add percentage formatting

### PR DESCRIPTION
**Pull request by codewhisperer**
screen shot 2018-10-22 at 16 24 04

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,17 @@ console.log(formatRate(1.23));
 // --> '1.23000'
 ```
 
+### Percentage formatting
+
+```javascript
+import { formatPercentage } from '@transferwise/formatting';
+
+console.log(formatPercentage(0.23456789));
```
Comment (oliverviljamaa) [inline README.md]: Let's have an example for the 1 decimal digit case as well, as that's the most controversial.

File: src/percentage/percentage.spec.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { formatPercentage } from '..';
+
+describe('Percentage formatting', () => {
+  it('formats percentage without zeros at the end', () => {
+    expect(formatPercentage(0.23)).toBe('23%');
```
Comment (oliverviljamaa) [inline src/percentage/percentage.spec.js]: I'd say there are 3 different _cases_ (no digits, 1 digit, more digits) – would be good to have separate tests for these as well, like here:
https://github.com/transferwise/formatting/blob/master/src/currency/currency.spec.js#L51

F.e. if we change the behaviour for 1 decimal digit, this test would start failing and the test description wouldn't tell us which case really fails.

File: src/percentage/percentage.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export function formatPercentage(value) {
+  return `${value.toFixed(4) * 100}%`;
```
Comment (Tankenstein) [inline src/percentage/percentage.js]: `toFixed` returns a string, and i don't like relying on javascript type coercion. Could you make what it's doing with the types a bit more explicit?

File: src/percentage/percentage.spec.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+describe('Percentage formatting', () => {
+  it('formats percentage without decimals', () => {
+    expect(formatPercentage(0.23)).toBe('23%');
+  });
+
+  it('formats percentage with one decimal', () => {
+    expect(formatPercentage(0.231)).toBe('23.1%');
+  });
+
+  it('formats percentage with two decimals', () => {
```
Comment (oliverviljamaa) [inline src/percentage/percentage.spec.js]:
```suggestion
  it('formats percentage with two or more decimals', () => {
```

**Pull request merged**